client/my-sites/exporter/export-card/index.jsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * External dependencies
+ */
+import { localize } from 'i18n-calypso';
+import flowRight from 'lodash/flowRight';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import SpinnerButton from './spinner-button';
+import FoldableCard from 'components/foldable-card';
+import analytics from 'lib/analytics';
+import Interval, { EVERY_SECOND } from 'lib/interval';
+import AdvancedSettings from './advanced-settings';
+import {
+	advancedSettingsFetch,
+	exportStatusFetch,
+	setPostType,
+	startExport,
+} from 'state/site-settings/exporter/actions';
+import {
+	shouldShowProgress,
+	getSelectedPostType,
+	isExporting,
+} from 'state/site-settings/exporter/selectors';
+
+class ExportCard extends Component {
+	componentWillMount() {
+		this.props.advancedSettingsFetch( this.props.siteId );
+	}
+
+	componentWillReceiveProps( newProps ) {
+		if ( newProps.siteId !== this.props.siteId ) {
+			this.props.advancedSettingsFetch( newProps.siteId );
+		}
+	}
+
+	trackExportClick = ( scope = 'all' ) => {
+		analytics.tracks.recordEvent( 'calypso_export_start_button_click', { scope } );
```

Comment:
We definitely want to avoid calling analytics functions from within React components. This breaks so many things. Thankfully, we have Redux actions available to replace it with!

```js
import { recordTracksEvent } from 'state/analytics/actions';

exportAll() {
	const {
		trackExportClick,
		startExport
	} = this.props;

	trackExportClick();
	startExport();
}

mapDispatchToProps = dispatch => ( {
	trackExportClick: ( scope = 'all' ) => dispatch( recordTracksEvent( 'calypso_export_start_button_click', { scope } ) ),
	…
} );
```